Gulpfile.js

```diff
@@ -9,30 +9,66 @@ ugifly = require('gulp-uglify');
 const libs = 'app/public/libs/'; 
 
 gulp.task('swigit', gulpSequence('lint', 'concat', ['minify-css', 'uglify']); 
+const gulp = require('gulp'),
```

Comment:
You're declaring gulp as a const, but the rest of the gulp dependencies aren't? If you're using strict mode, this should be throwing an error.